WordPressCom-Stats-iOS/Services/WPStatsServiceRemote.m

```diff
@@ -1127,7 +1133,7 @@ - (AFHTTPRequestOperation *)operationForAuthorsForDate:(NSDate *)date
             for (NSDictionary *post in posts) {
                 StatsItem *postItem = [StatsItem new];
                 postItem.itemID = [post numberForKey:@"id"];
-                postItem.label = [[post stringForKey:@"title"] stringByDecodingXMLCharacters];
+                postItem.label = [[[post stringForKey:@"title"] stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]] stringByDecodingXMLCharacters];
```

Comment:
I kinda want to make a helper method for these, passing in `[postDict stringForKey:@"title"]` and returning the formatted title. What do ya think?